viite-backend/base/src/test/scala/fi/vaylavirasto/viite/model/AddrMRangeSpec.scala

```diff
@@ -78,9 +78,71 @@ class AddrMRangeSpec extends AnyFunSuite with Matchers {
 
 
   // ----------------------------------- Connectivity checks -----------------------------------
+
+  test("Test AddrMRange.isSameAs(AddrMRange): Returns true, if the compared AddrMRanges have same start, and end. Else false") {
+    //                                                                           100    300 // comparison range
+    //AddrMRange "A"                      AddrMRange "B"                  //      |AAAAAA|      //
+    AddrMRange(100,300).isSameAs(AddrMRange(100,300)) shouldBe true       //      |BBBBBB|      // Are the same
+
+    // different either value
+    AddrMRange(100,300).isSameAs(AddrMRange(  0, 99)) shouldBe false      // BBBB |      |      // Do not even touch
+    AddrMRange(100,300).isSameAs(AddrMRange(301,400)) shouldBe false      //      |      | BBBB // Do not even touch
+    AddrMRange(100,300).isSameAs(AddrMRange(  0,100)) shouldBe false      // BBBBB|      |      // Only single point in common
+    AddrMRange(100,300).isSameAs(AddrMRange(300,400)) shouldBe false      //      |      |BBBBB // Only single point in common
+    AddrMRange(100,300).isSameAs(AddrMRange(101,300)) shouldBe false      //      | BBBBB|      // Only single point in common
```

Comment:
Kahdella vikalla tuo kommentti copy paste virhe?

Reply:
Eei, kyl ne ihan oikein on. Oisko näin selkeämpi:

    AddrMRange(100,300).isSameAs(AddrMRange(101,300)) shouldBe false      //      |   BBB|      // Only single point in common
    AddrMRange(100,300).isSameAs(AddrMRange(100,299)) shouldBe false      //      |BBB   |      // Only single point in common

Reply:
Eeeiku nii single point! Joo!
Öh. Noloa :D
Kiitos!

Reply:
"Nearly the same but not exactly"